system/jlib/jthread.cpp

```diff
@@ -553,7 +553,91 @@ unsigned getThreadCount()
 }
 
 // CThreadedPersistent
+#ifdef NEW_THREADEDPERSISTENT
+class GlobalThreadPool : public CInterfaceOf<IThreadFactory>
```

Comment:
The JIRA mentioned that the global thread pool was to gradually shrink over time if threads went unused. I don't see any code here that would do that.